scripts/demo/template-projects/templates_in_master.json

```diff
@@ -64,19 +66,21 @@
         ],
         "outputs": {},
         "progress": 85,
+        "thumbnail": "",
         "position": {
           "x": 400,
           "y": 175
         }
       }
-    }
+    },
+    "bootOptions": "{}"
```

Comment:
I do agree with @pcrespov here. I find pretty painful when we need to insert empty values instead of nothing. the payload shall be empty = no values. it will spare some bytes in the transfer (especially of Projects which are already rather big.)

Reply:
by "painful"I mainly mean problem is copying data from one data structure to another and expecting the validation not to fail!